### ISSUE TITLE
Can't use stylelint-config-prettier

### ISSUE THREAD

**Issue by trobrock** [gilbarbara/codeclimate-stylelint#39]
When running in CodeClimate's hosted environment I get:

```
•• Timing: .engineConfig: 0.089s
Error: Could not find "stylelint-prettier/recommended". Do you need a `configBasedir`?
See our documentation at https://docs.codeclimate.com/docs/stylelint for more information.
```
Is there something special I need to do to use this plugin in the hosted environment?

**Comment by trobrock** [comment]
This was an issue with my config, sorry for the early post I should've ran it locally first.

**Comment by trobrock** [comment]
Sorry, I take that back. My config was bad, but now it runs locally and am still seeing the error in the hosted environment:

```
•• Timing: .engineConfig: 0.083s
Error: Could not find "stylelint-config-prettier". Do you need a `configBasedir`?
See our documentation at https://docs.codeclimate.com/docs/stylelint for more information.
```

.stylelintrc
```json
{
  "extends": ["stylelint-config-standard", "stylelint-config-prettier"],
  "plugins": ["stylelint-order"],
  "rules": {
    "order/order": [
      "custom-properties",
      "declarations"
    ],
    "order/properties-alphabetical-order": true
  }
}
```

.codeclimate.yml
```yaml
plugins:
  rubocop:
    enabled: true
  eslint:
    enabled: false
  stylelint:
    enabled: true
  scss-lint:
    enabled: true
  bundler-audit:
    enabled: true
  fixme:
    enabled: true

exclude_patterns:
  - ".rubocop*.yml"
  - "spec/"
  - "node_modules/"
  - "!spec/support/helpers"
  - "config/"
  - "db/"
  - "spec/"
  - "vendor/"
  - "coverage/"
  - "bin/"
  - ".idea/"
  - "Guardfile"
```

**Comment by gilbarbara** [comment]
hey @trobrock 

Codeclimate.com is running a really old version of this engine, they need to update it.

I've released a new version (2.3.0) with some updates

**Comment by lougreenwood** [comment]
@gilbarbara I also just hit this problem, can you suggest how we can contact code climate and request that they update the engine?
